misc/docs/source/tutorial/code_snippets/stationxml_custom_tags.rst

```diff
@@ -0,0 +1,167 @@
+.. _stationxml-extra:
+
+=============================================================================
+Handling custom defined tags in StationXML with the Obspy Inventory
+=============================================================================
+
+StationXML allows use of custom elements in addition to the 'usual' information
+defined by the StationXML standard. It allows *a)* custom namespace attributes to
+StationXML namespace tags and *b)* custom namespace subtags to StationXML namespace
+elements.
+ObsPy can handle both basic custom tags in the Inventory object's Network, Station, and Channel objects (*a*) and custom
```

Comment:
Yes you can add custom tags/attributes on all "base" elements, as demonstrated in the test test_write_with_extra_tags_and_read. There are still some limitations however, like adding custom tags to certain sub-elements. For example, you currently can't add custom tags to the InputUnits sub element of the Stage Coefficients element. It seemed non-trivial to add support for custom tags on these sub-elements.